server/internal/adapter/gql/resolver_mutation_field.go

```diff
@@ -125,3 +128,46 @@ func (r *mutationResolver) DeleteField(ctx context.Context, input gqlmodel.Delet
 		FieldID: input.FieldID,
 	}, nil
 }
+
+func (r *mutationResolver) UpdateFieldsOrder(ctx context.Context, input gqlmodel.UpdateFieldsOrderInput) (*gqlmodel.FieldsPayload, error) {
+	params, err := util.TryMap(input.FieldsOrder, func(fo *gqlmodel.FieldOrder) (interfaces.UpdateFieldsOrderParam, error) {
+		fid, err := gqlmodel.ToID[id.Field](fo.FieldID)
+		if err != nil {
+			return interfaces.UpdateFieldsOrderParam{}, err
+		}
+		return interfaces.UpdateFieldsOrderParam{FieldId: fid, Order: fo.Order}, nil
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	mId, err := gqlmodel.ToID[id.Model](input.ModelID)
+	if err != nil {
+		return nil, err
+	}
+
+	ms, err := usecases(ctx).Model.FindByIDs(ctx, []id.ModelID{mId}, getOperator(ctx))
+	if err != nil || len(ms) != 1 || ms[0].ID() != mId {
+		if err == nil {
+			return nil, errors.New("not found")
+		}
+		return nil, err
+	}
+	m := ms[0]
+
+	s, err := usecases(ctx).Schema.FindByID(ctx, m.Schema(), getOperator(ctx))
```

Comment:
IMO there is no need to load the schema here, you can just pass the `m.Schema()` to `UpdateFieldsOrder`